src/aca_main.cpp

```diff
@@ -187,14 +205,22 @@ int main(int argc, char *argv[])
   if (g_rpc_protocol == EMPTY_STRING) {
     g_rpc_protocol = UDP_PROTOCOL;
   }
+  if (g_ofctl_command == EMPTY_STRING) {
+    g_ofctl_command = OFCTL_COMMAND;
+  }
+  if (g_ofctl_target == EMPTY_STRING) {
+    g_ofctl_target = OFCTL_TARGET;
+  }
+  
+  // async_grpc_server = new Aca_Async_GRPC_Server();
+  // async_grpc_server_thread =
+  //        new std::thread(std::bind(&Aca_Async_GRPC_Server::Run, async_grpc_server));
 
-  async_grpc_server = new Aca_Async_GRPC_Server();
```

Comment:
please don't comment out the grpc code because it will disable the ability to receive goalstate message via grpc

Reply:
those comments are removed now